scss/_variables.scss

```diff
@@ -1078,8 +1078,8 @@ $modal-scale-transform:             scale(1.02) !default;
 //
 // Define alert colors, border radius, and padding.
 
-$alert-padding-y:                   .75rem !default;
-$alert-padding-x:                   1.25rem !default;
+$alert-padding-y:                   $spacer * .75 !default;
```

Comment:
Why keeping an exotic value here? `.5rem` or `1rem` looks good too, don't they?
This would be the last value out of the spacing scale.

Reply:
Vertical padding I care less about syncing up, but will look at this again to see how they look when cut down to `.5` or up to `1`.

Reply:
Done! Went with `1rem`. Thanks!